File: lib/model/word_section.dart
import 'package:backend/backend.dart';
import 'package:backend/model/text.dart';

class WordSection extends ManagedObject<_WordSection> implements _WordSection {}

class _WordSection {
  @primaryKey
  int id;

  @Column(unique: false)
  String word;

  @Column(unique: false)
  bool isPressed;

  @Relate(#words)
  Text text;
}

File: DB/WordSection.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace tcc_lang_backend.DB
{
    public class WordSection
    {
        [Key] public int Id { get; set; }
        public string Word { get; set; }
        public bool IsPressed { get; set; }
        public Text Text { get; set; }
    }
}